tools/rush-update/src/types.d.ts

```diff
@@ -5,5 +5,5 @@ declare module 'npm-check-updates' {
     upgrade?: boolean,
   }
 
-  function run(params: NCUParams): Record<string, string[]>;
+  function run(params: NCUParams): Promise<Record<string, string[]>>;
```

Comment:
This was a bug in the old type definitions, right?

Reply:
Yep! The implementation correctly awaited.